File: DSA/Vritika_590012956/Day_15_Question_3.py
numbers = input("Enter numbers separated by spaces: ")
arr = [int(x) for x in numbers.split()]

# Bubble sort 
n = len(arr)
for i in range(n):
    for j in range(n - 1):
        if arr[j] > arr[j + 1]:
            # Swap elements
            arr[j], arr[j + 1] = arr[j + 1], arr[j]

print("Sorted list:", arr)

File: DSA/Shreya_Goyal_590015491/Day_15_Question_3.c
#include <stdio.h>

int main() {
    int arr[8], i, j, temp;

    printf("Enter numbers:\n");
    for(i=0; i<8; i++) {
        scanf("%d", &arr[i]);
    }

    for(i=0; i < 8 - 1; i++) {
        for(j=0; j < 8 - 1 - i; j++) {
            if(arr[j] > arr[j + 1]) {
                temp = arr[j];
                arr[j] = arr[j + 1];
                arr[j+1] = temp;
            }
        }
    }

    printf("Sorted array:\n");
    for(i = 0; i < 8; i++) {
        printf("%d ", arr[i]);
    }

    return 0;
}
